.github/pull_request_template.md

```diff
@@ -1,48 +1,15 @@
-* what this PR does:
-* why this is needed:
-* context for anyone reading this in 6 months or years:
-* [ ] added screenshots/videos for if applicable
+## What this PR does
 
-fix #{{issue-no}}
-
-## Review Protocol
-
-* [ ] 🤯 code is understandable
-* [ ] 🧪 code is covered by specs
-* [ ] 🚀 no performance issues found
-* [ ] 👮 no security issues found
-
-## QA Protocol
-
-* [ ] 🐭 manually clicked through the feature or manually called the API
-* [ ] 🖼️ everything looks ok visually (mobile and desktop screen sizes)
-* [ ] 🔑 no A11y issues found 
-* [ ] flows tested:
-  * 1
-  * 2
-  * 3
-
-## QA
+* Description; why this is needed; context for anyone reading this in 6 months or years
```

Comment:
what would "description" entail?

Reply:
What happens in this PR, summary of the changes.

e.g.
"replaced all icon buttons with a new link helper"